src/index.js

```diff
@@ -408,15 +408,39 @@ function tab({shift = false, focusTrap = document} = {}) {
 
       return diff === 0 ? a.idx - b.idx : diff
     })
+    .map(({el}) => el)
+
+  if (shift) orderedElements.reverse()
+
+  // keep only the checked or first element in each radio group
+  const prunedElements = []
+  for (const el of orderedElements) {
+    if (el.type === 'radio' && el.name) {
+      const replacedIndex = prunedElements.findIndex(
+        ({name}) => name === el.name,
+      )
+
+      if (replacedIndex === -1) {
+        prunedElements.push(el)
+      } else if (el.checked) {
+        prunedElements.splice(replacedIndex, 1)
+        prunedElements.push(el)
+      }
+    } else {
+      prunedElements.push(el)
+    }
+  }
+
+  if (shift) prunedElements.reverse()
```

Comment:
Is it already reversed since you reversed the ordered elements above?

Reply:
Yes. I'm reversing it twice (before and after the pruning) when using `shift` because I couldn't figure out a clean way to make the pruning logic reverse directions without duplicating it.

Reply:
Nope that's cool just checking